src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const properties = sourceString
+    .split(';')
+    .map(property => property.split(':'))
+    .filter(property => property.length === 2);
+  const callback = (object, property) => {
+    return {
+      ...object,
+      [property[0].trim()]: property[1].trim(),
+    };
+  };
```

Comment:
Уникай загальних назв змінних, типу `callback` або `object`
```suggestion
  const addProperty = (styles, property) => {
    return {
      ...object,
      [property[0].trim()]: property[1].trim(),
    };
  };
```